compiler/quilt/test/test_command.py

```diff
@@ -411,14 +448,40 @@ def test_user_disable_no_auth(self):
         with self.assertRaises(command.CommandException):
             command.disable_user('bob')
         pass
-    
+
+    def test_user_disable_empty(self):
+        self._mock_method('disable', status=400, message="Username is not valid")
+        with assertRaisesRegex(self, command.CommandException, "Username is not valid"):
+            command.disable_user('')
+
+        self._mock_method('disable', status=400, team='qux', message="Username is not valid")
+        with assertRaisesRegex(self, command.CommandException, "Username is not valid"):
+            command.disable_user('', team='qux')
+
+    def test_user_disable_unknown(self):
```

Comment:
pls also test disable a user from a non-existent team
pls also test disable a non-existent user
pls also test disable an already-disabled user
pls also test disable an already-deleted user

at the top of the file, please create a table (in text!) of these cross-product test cases, and then make sure you've got them all covered.